EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -46,14 +59,18 @@ public function attachRefreshToken(AuthenticationSuccessEvent $event)
 
         $refreshTokenString = RequestRefreshToken::getRefreshToken($request);
 
+        $user->setLastLogin(new \DateTime());
```

Comment:
This will break if app user doesn't have this method as symfony UserInterface doesn't have this method